### PR TITLE
CS598DHL Extra Credit - Added The Automated Cardiac Diagnosis Challenge (ACDC) dataset.

### PR DESCRIPTION
**Pull request by sherrylinice**
This PR is for CS598 SP24 Extra Credit: Added the the Automated Cardiac Diagnosis Challenge (ACDC) dataset. Contributed by Sherry Li (NetID: xuehail2). 
URL: https://www.creatis.insa-lyon.fr/Challenge/acdc/databases.html

The dataset contains the NIfTI-1 cardiac cine-MRI data. Including:

- Training set: 
   - 20 healthy patients;
   - 20 patients with previous myocardial infarction;
   - 20 patients with dilated cardiomyopathy;
   - 20 patients with an hypertrophic cardiomyopathy;
   - 20 patients with abnormal right ventricle;
 
- Test set: 
   - 10 healthy patients;
   - 10 patients with previous myocardial infarction;
   - 10 patients with dilated cardiomyopathy;
   - 10 patients with an hypertrophic cardiomyopathy;
   - 10 patients with abnormal right ventricle;

Citation:
O. Bernard, A. Lalande, C. Zotti, F. Cervenansky, et al.
"Deep Learning Techniques for Automatic MRI Cardiac Multi-structures Segmentation 
and Diagnosis: Is the Problem Solved ?"
in IEEE Transactions on Medical Imaging, vol. 37, no. 11, pp. 2514-2525, Nov. 2018
doi: 10.1109/TMI.2018.2837502

### CODE REVIEW

**Review by ycq091044 — APPROVED** [review]
I will merge into the course-pr branch and test it locally.
